_posts/en/newsletters/2020-11-25-newsletter.md

```diff
@@ -0,0 +1,146 @@
+name: 2020-11-25-newsletter
+slug: 2020-11-25-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter announces a website that tracks miner support for
+taproot activation and includes our regular sections with popular
+questions and answers from the Bitcoin StackExchange, announcements of
+releases and release candidates, and notable changes to popular Bitcoin
+infrastructure software.
+
+## Action items
+
+*None this week.*
+
+## News
+
+- **Website listing miner support for taproot activation:** Bitcoin
+  mining pool [Poolin][] has created [a website][taprootactivation.com]
+  to help track miner support for activating
+  [taproot][topic taproot] (including [schnorr signatures][topic schnorr
+  signatures] and [tapscript][topic tapscript]).  As of this writing,
+  all pools listed on the site---who represent more than half of the
+  current network hash rate---have indicated that they will support
+  activation.  If this is reflective of the opinions of the remaining
+  pools, then it should be easy to accomplish the activation of taproot
+  after its implementation has been released and adopted by a moderate
+  number of users.  For additional information about the creation of the
+  website, see Bitcoin Magazine's [article][btcmag ta.com].
+
+- **New research and development organization announced:** [Brink][], a
+  new organization for "funding, education, and mentoring" of Bitcoin
+  contributors was [announced][brink tweet] on Twitter this week.  The
+  organization is seeking applicants for both its [fellowship
+  program][brink fellowship] for helping newer contributors break into
+  full-time Bitcoin development and its [grant program][brink grant] for
+  funding established contributors.  (Note: Brink was founded by Optech
+  contributors and received initial funding from John Pfeffer and Wences
+  Casares, who also provided initial funding for Optech.)
+
+## Selected Q&A from Bitcoin StackExchange
+
+*[Bitcoin StackExchange][bitcoin.se] is one of the first places Optech
+contributors look for answers to their questions---or when we have a
+few spare moments to help curious or confused users.  In
+this monthly feature, we highlight some of the top-voted questions and
+answers posted since our last update.*
+
+{% comment %}<!-- https://bitcoin.stackexchange.com/search?tab=votes&q=created%3a1m..%20is%3aanswer -->{% endcomment %}
+{% assign bse = "https://bitcoin.stackexchange.com/a/" %}
+
+- [What is fuzz testing?]({{bse}}99955)
+  Michael Folkson summarizes [fuzz testing][fuzzing wikipedia], a technique of feeding a program like
+  Bitcoin Core varying malformed inputs for purposes of finding bugs.
+
+- [Adding instead of concatenating hashes in Merkle trees]({{bse}}100098)
+  Pieter Wuille breaks down a variety of issues to consider regarding the idea
+  of combining hashes in a merkle tree "using a commutative operation such as
+  addition instead of concatenating them".
```

Comment:
them." -> them."

Reply:
I would have written it the way it is, since not the full sentence is a quote, but just the last part.

Reply:
Yes, but afaik it still needs to be written with the quote after the full stop.